commands/command.go

```diff
@@ -209,6 +211,15 @@ func (c *BaseCommand) run(args []string) error {
 		c.storage = storage.New(fileStrategy)
 	}
 
+	user, err := c.User()
+	if err != nil {
+		return err
+	}
+
+	c.TelemetryService.Setup(c.Name)
```

Comment:
`c.TelemetryService.Setup(c.Name)` can still be moved up higher in this code chain (in case an `error` is returned from `run() error` here early, we'll at least get "command name" in some of the event properties

I'd place it b/w the version check and the storage check in this method